NEWS.md

```diff
@@ -1,5 +1,16 @@
 # Version 0.0
 
+## Changes in 0.0.4
+
+- Add *alabaster* `saveObject()` and `readObject()` methods as well as
+  `saveMsObject()` and `readMsObject()` methods with `AlabasterParam` param for
+  `MsBackendMzR`, `Spectra`, `MsExperiment` and `XcmsExperiment` objects.
+
+## Changes in 0.0.3
```

Comment:
oh sorry I forgot to do this !!

Reply:
no problem. it's also more for me to remember what was added by which version :)